### PR TITLE
Fix #793

### LINKED ISSUE #793: App init does not work
error log:
```
Class 'OCA\MyApp\AppInfo\App' not found at /var/www/html/custom_apps/myapp/lib/AppInfo/Application.php#5
```

https://github.com/nextcloud/documentation/blame/master/developer_manual/app/init.rst#L64-L78

appinfo/app.php
```
<?php

$app = new \OCA\MyApp\AppInfo\Application();
```
lib/AppInfo/Application.php
```
<?php
namespace OCA\MyApp\AppInfo;

class Application extends App {
}
```

documentation doesn't work?

### PR DESCRIPTION
**Pull request by lil5**
Add forgotten declaration of App

### CODE REVIEW

**Comment by lil5** [comment]
Don't think it's the right class :tired_face:

File: developer_manual/app/init.rst [hunk 1]
```diff
@@ -68,6 +68,8 @@ lib/AppInfo/Application.php
 
     <?php
     namespace OCA\MyApp\AppInfo;
+    
+    use OCP\App;
```
Comment (juliushaertl) [inline developer_manual/app/init.rst]: it is `OCP\AppFramework\App`

**Comment by juliushaertl** [comment]
@lil5 Thanks for fixing this. The correct class is actually `OCP\AppFramework\App`. :wink:

**Comment by MorrisJobke** [comment]
stable13 277b1254 58fb0edb

**Pull request merged**